tests/App/config/yves.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
 use Spryker\Shared\Application\ApplicationConstants;
+use Spryker\Shared\Router\RouterConstants;
 
-$config[ApplicationConstants::YVES_SSL_ENABLED] = false;
+$config[RouterConstants::YVES_IS_SSL_ENABLED] = false;
+$config[RouterConstants::YVES_SSL_EXCLUDED_ROUTE_NAMES] = ['/'];
 $config[ApplicationConstants::HOST_YVES] = getenv('HOST_YVES') ?: 'localhost';
-$config[ApplicationConstants::PORT_YVES] = getenv('PORT_YVES') ?: '8080';
+$config[ApplicationConstants::PORT_YVES] = getenv('PORT_YVES') ?: '8085';
```

Comment:
Not 100% sure about this - I was using 8085 for yves and 8086 for zed to match `.travis.yml` but maybe this is completely wrong.